templates/workers/worker-horizontalpodautoscaler.yaml

```diff
@@ -1,34 +0,0 @@
-################################
-## Airflow Worker HorizontalPodAutoscaler
-#################################
-{{- if (and .Values.workers.autoscaling.enabled (eq .Values.airflow.executor "CeleryExecutor")) }}
-apiVersion: autoscaling/v2beta1
-kind: HorizontalPodAutoscaler
-metadata:
-  name: {{ .Release.Name }}-worker
-  labels:
-    tier: airflow
-    component: worker-horizontalpodautoscaler
-    release: {{ .Release.Name }}
-    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
-    heritage: {{ .Release.Service }}
-{{- with .Values.airflow.labels }}
-{{ toYaml . | indent 4 }}
-{{- end }}
-spec:
-  scaleTargetRef:
-    apiVersion: apps/v1
-    kind: StatefulSet
-    name: {{ .Release.Name }}-worker
-  minReplicas: {{ .Values.workers.autoscaling.minReplicas }}
```

Comment:
You need to remove this stuff from values as well.

Reply:
Keda was never shipped as a feature in Astronomer. This is effectively removing dead code that was always dead. I suppose it is possible that folks are using this chart outside of astronomer though...

Reply:
README , featuer flags and values.yaml cleanup with KEDA and hpa is done